### PR TITLE
create a modal that opens when unsaved changes are on a form

### PR DESCRIPTION
**Pull request by toastynerd**
Looks like I had accidentally committed a vim temp file on a previous merge, this also gets rid of that. @joshreeves @shenst1

### CODE REVIEW

File: vendor/assets/javascripts/changed_form_modal.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var NR = Cmsify.NestedResource.prototype;
+var CFM = Cmsify.ChangedFormModal = function(el) {
+  this.$el = $(el);
+  this.form = this.$el.find('form');
+  this.modal = UIkit.modal(this.$el.find('#js-unsaved-modal'));
```
Comment (shenst1) [inline vendor/assets/javascripts/changed_form_modal.js]: where is the ```#js-unsaved-modal``` coming from in the dom? Also, use a class for dom targeting with js

Comment (toastynerd) [reply vendor/assets/javascripts/changed_form_modal.js]: Line 5 of this: https://github.com/plyinteractive/spr-admin/pull/88/files#diff-319d6fb7182fce1d33bc458c66fa792e it's the modal that pops up when you have unsaved changes as per this design: https://projects.invisionapp.com/d/main#/console/9694214/216096470/preview I can switch it over to a class.

Comment (joshreeves) [reply vendor/assets/javascripts/changed_form_modal.js]: Yeah, use classes so multiple instances can exist on the page.

**Review by joshreeves — COMMENTED** [review]
LGTM 🚢

**Pull request merged**